<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddDevice</class>
 <widget class="QWidget" name="DialogAddDevice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="surnameLineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="nameLineEdit"/>
     </item>
     <item>
      <widget class="QLineEdit" name="lastnameLineEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="nextButton">
     <property name="text">
      <string>Наступний</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Попередній</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
